Sum con (#32)
chhoumann/P6: sections/summary.tex

Finally, the models were trained using arbitrary settings for the batch sizes, hyperparameters, learning and dropout rates.

\subsection{Conclusion}

> Having used weather data gathered from different weather sensors we have trained and compared different machine learning models for time series forecasting.

Reviewer: ```suggestion
Having used weather data gathered from different weather sensors, we have trained and compared different machine learning models for time series forecasting.
```

> The focus of the paper was to show that the transformer machine learning model would outperform previous state-of-the-art statistical and machine learning techniques. This we have shown to be the case. 

Reviewer: ```suggestion
The focus of this paper was to show that the transformer machine learning model would outperform previous state-of-the-art statistical and machine learning techniques. 
We have presented new evidence for this to be the case.
```

> Additionally, we developed a web application to show the results of the trained models and the user is able to interact with this web application by using a dropdown to select a model and a dataset. This will then make temperature predictions.

Reviewer: ```suggestion
Additionally, we developed a web application to show the results of the trained models, and the user is able to interact with this web application by using a dropdown to select a model and a dataset. This will then show the temperature predictions.
```
